### PR TITLE
Fix CICS Dump Transaction Command

### PR DESCRIPTION
**Pull request by KutluOzel-b**
## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by KutluOzel-b** [comment]
@chacebot @mm-broadcom @ilidio-lopes Can you please review ?

File: server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSDumpTransactionOptionsCheckUtility.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    put(CICSLexer.PROGRAM, ErrorSeverity.ERROR);
+                    put(CICSLexer.TERMINAL, ErrorSeverity.ERROR);
+                    put(CICSLexer.TABLES, ErrorSeverity.ERROR);
+                    put(CICSLexer.FCT, ErrorSeverity.ERROR);
+                    put(CICSLexer.PCT, ErrorSeverity.ERROR);
+                    put(CICSLexer.PPT, ErrorSeverity.ERROR);
+                    put(CICSLexer.SIT, ErrorSeverity.ERROR);
+                    put(CICSLexer.TCT, ErrorSeverity.ERROR);
+                    put(CICSLexer.DUMPID, ErrorSeverity.ERROR);
+                }
```
Comment (ilidio-lopes) [inline server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSDumpTransactionOptionsCheckUtility.java:60]: Warning for all unparametrized options

Comment (ilidio-lopes) [reply server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSDumpTransactionOptionsCheckUtility.java:60]: and conflicts

Comment (KutluOzel-b) [reply server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSDumpTransactionOptionsCheckUtility.java:60]: should adressed at :[df409f7](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2577/commits/df409f736fab68af7185c967043caa596e5e40bc)

**Review by ilidio-lopes — APPROVED** [review]

**Pull request merged**